src/ApiHandler.ts

```diff
@@ -181,7 +181,7 @@ export default class ApiHandler {
 				reserved,
 				miscFrozen,
 				feeFrozen,
-				locks,
+				locks: Array.isArray(locks) ? locks : [ locks ],
```

Comment:
Noting this is actually incorrect...the desire was to make sure that `locks.reasons` is always an array.